addons/packages/app-toolkit/0.1.0/README.md

```diff
@@ -69,57 +84,81 @@ excluded_packages:
   - cert-manager.community.tanzu.vmware.com
 ```
 
-### Local Docker installation without load balancer
-
-To deploy contour with a ClusterIP configuration, and Knative to serve at localhost, add these settings to a `values.yaml` file. Also, add a Docker registry for kpack to store produced container images.
-
-```yaml
-contour:
-  envoy:
-    service:
-      type: ClusterIP
-    hostPorts:
-      enable: true
-
-knative_serving:
-  domain:
-    type: real
-    name: 127-0-0-1.sslip.io
-
-kpack:
-  kp_default_repository: https://index.docker.io/v1/
-  kp_default_repository_username: [your username]
-  kp_default_repository_password: [your password]
-```
 
 ## Usage Example
 
-This example illustrates creating a simple Spring Boot web app on a locally deployed TCE cluster using Docker.
+This example illustrates creating a simple Spring Boot web app on the locally deployed TCE cluster using Docker.
 
-1. Begin with Docker and the Tanzu CLI installed on your development machine.
-
-  For best performance, configure Docker with at least 8GB of RAM.
+1. Begin with Docker and the Tanzu CLI installed on your development machine and ensure you have the Carvel tools installed as well as we will be leveragig `ytt` in the examples below. For best performance, configure Docker with at least 8GB of RAM and that you have plenty of free space on the local filesystem that Docker is using.
```

Comment:
@krisapplegate why do we need 8GB of RAM? Have we validated that?

Reply:
It was in the previous doc, I just ported it forward.

Reply:
Let's remove that please. We don't know if that's true.